test/test_party_controller.py

```diff
@@ -38,6 +38,82 @@ def test_get_business_by_id_returns_correct_representation(self):
         self.assertEqual(response.get('name'), mock_business.get('name'))
         self.assertEqual(response.get('trading_as'), 'Tradstyle-1 Tradstyle-2 Tradstyle-3')
 
+    def test_get_businesses(self):
+        mock_business_1 = MockBusiness() \
+            .attributes(source='test_get_business_by_ids_returns_correct_representation') \
+            .as_business()
+
+        mock_business_2 = MockBusiness() \
+            .attributes(source='test_get_business_by_ids_returns_correct_representation') \
+            .as_business()
+
+        party_id_1 = self.post_to_businesses(mock_business_1, 200)['id']
+        party_id_2 = self.post_to_businesses(mock_business_2, 200)['id']
+        self.assertNotEquals(party_id_1, party_id_2)
```

Comment:
This seems a bit of overkill

Reply:
Maybe, but if someone changes how the ids are generated and they suddenly become the same the rest of the test is pointless. I don't see the harm in having it.